recipe/LICENSE

```diff
@@ -200,232 +200,35 @@
    See the License for the specific language governing permissions and
```

Comment:
To be removed as the tarball contains a LICENSE file (previously missing on pypi).